Source/ThirdParty/ANGLE/src/libANGLE/Context.cpp

```diff
@@ -895,6 +895,7 @@ egl::Error Context::onDestroy(const egl::Display *display)
 
     releaseShaderCompiler();
 
+    mState.ensureNoPendingLink(this);
```

Comment:
Per feedback on the upstream patch https://chromium-review.googlesource.com/c/angle/angle/+/4957715 , Shabi and I think this should be moved to the top of the file, before the context is removed from the share group.